wagtail/wagtailimages/tests/test_image_operations.py

```diff
@@ -1,4 +1,4 @@
-import unittest
+from django.test import TestCase
```

Comment:
This change may well be unnecessary. I had to update these tests as they were originally creating Image objects in module scope (i.e. before the database was initialised), and that's no longer possible since we have to do a DB lookup to find the default (root) collection. It took a few hours to track this down as the root cause, and changing `unittest.TestCase` to `django.test.TestCase` was just my first pass at trying to fix it.